index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kevin-Koome-Portfolio</title>
+</head>
+<body>
+    <div class="container">
+            <nav class="Top-Navigation">
+                <div class="Item-Navigation">
+                    <a href="#" class="Contact">Contact</a>
+                    <a href="#" class="About">About</a>
+                    <a href="#" class="Portfolio">Portfolio</a>
+                </div>
+            </nav>
+            <div class="Name">
+                <p class="Koome">Kevin Koome</p>
+            </div>
+            <header class="header">
```

Comment:
- [ ] Kindly center the content.
![Captura de pantalla_2021-10-20_21-55-38](https://user-images.githubusercontent.com/53111464/138203740-0300c8ad-2c08-4567-8b9f-ee96dfa8d757.png)
- Screen size Pixel2 width: 411px
- [ ] Kindly use font-size, font-family, font-weight values same as the Figma design.
- [ ] The section has the margins (top, bottom, left, right) defined in the design
- [ ] Add background images just like the figma design.
![Captura de pantalla_2021-10-20_22-09-45](https://user-images.githubusercontent.com/53111464/138204889-558ccaaf-a01b-4a33-937a-66a004062c5b.png)